Sources/SpeziOnboarding/Resources/Localizable.xcstrings

```diff
@@ -18,6 +21,7 @@
       }
     },
     "CONSENT_EXPORT_ERROR_DESCRIPTION" : {
+      "extractionState" : "stale",
```

Comment:
We should remove some of the stale localizations.

Reply:
Thanks, totally missed that!